### PR TITLE
My priorities upgrade

### PR DESCRIPTION
**Pull request by Revenaunt**
Animated version of My Priorities.

### CODE REVIEW

**Comment by Revenaunt** [comment]
@bradmontgomery there are still a couple of things I need to look into, sometimes actions don't display created triggers when added through My Priorities. I wonder if that has to do with the way data is handled within the application.

File: src/main/java/org/tndata/android/compass/activity/MyPrioritiesActivity.java [hunk 1]
```diff
@@ -62,10 +82,38 @@ private void swapFragments(int index, boolean addToStack, Category category){
             if (addToStack){
                 mFragmentStack.add(fragment);
             }
-            getFragmentManager().beginTransaction().replace(R.id.base_content, fragment).commit();
+            if (index == CATEGORIES){
+                mToolbar.setTitle("My Priorities");
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/activity/MyPrioritiesActivity.java]: lets put this in `strings.xml` somewhere. There may already be a value for this (e.g. the title of the nav drawer menu).

File: src/main/java/org/tndata/android/compass/adapter/MyPrioritiesGoalAdapter.java [hunk 2]
```diff
@@ -69,30 +95,147 @@ public long getItemId(int position){
...
+            behaviorView.setLeftPadding(20);
+            behaviorView.getTextView().setText(behavior.getTitle());
+            if (behavior.getIconUrl() != null){
+                ImageLoader.loadBitmap(behaviorView.getImageView(), behavior.getIconUrl(), false);
+            }
+            behaviorView.setOnClickListener(holder);
+
+            //The view is added to the goal's offspring
+            holder.offspring.addView(behaviorView);
+            Log.d("BehaviourActions", behavior.getActions().size() + "");
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/adapter/MyPrioritiesGoalAdapter.java]: s/Behaviour/Behavior/  :us:   :)

File: src/main/java/org/tndata/android/compass/ui/PriorityItemView.java [hunk 3]
```diff
@@ -9,34 +9,61 @@
...
 
+    /**
+     * Creates the layout of the view.
+     */
     private void initViews(){
         setOrientation(LinearLayout.HORIZONTAL);
-        setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT,
-                ViewGroup.LayoutParams.WRAP_CONTENT));
+        setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
+                getPixels(35)));
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/ui/PriorityItemView.java]: Will this be OK across different screen sizes? Should we put the `35` in `dimens.xml`?

**Comment by bradmontgomery** [comment]
@Revenaunt  :+1: feel free to merge.

**Pull request merged**